network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreExtensionsIT.java

```diff
@@ -483,6 +483,61 @@ private Network createExtensionsNetwork(NetworkFactory networkFactory) {
+            assertEquals(10, connectablePositionload.getFeeder().getOrder().orElseThrow().intValue());
+            assertEquals(ConnectablePosition.Direction.BOTTOM, connectablePositionload.getFeeder().getDirection());
+
+            ConnectablePosition connectablePositionTwt2 = twt2.getExtension(ConnectablePosition.class);
+            assertNotNull(connectablePositionTwt2);
+            connectablePositionTwt2.getFeeder1().setName("TWT2Modified.1", 1).setOrder(20, 1).setDirection(ConnectablePosition.Direction.BOTTOM, 1);
+            connectablePositionTwt2.getFeeder2().setName("TWT2Modified.2", 2).setOrder(20, 2).setDirection(ConnectablePosition.Direction.BOTTOM, 2);
+            assertEquals("TWT2Modified.1", connectablePositionTwt2.getFeeder1().getName().orElseThrow());
+            assertEquals(20, connectablePositionTwt2.getFeeder1().getOrder().orElseThrow().intValue());
+            assertEquals(ConnectablePosition.Direction.BOTTOM, connectablePositionTwt2.getFeeder1().getDirection());
+            assertEquals("TWT2Modified.2", connectablePositionTwt2.getFeeder2().getName().orElseThrow());
+            assertEquals(20, connectablePositionTwt2.getFeeder2().getOrder().orElseThrow().intValue());
+            assertEquals(ConnectablePosition.Direction.BOTTOM, connectablePositionTwt2.getFeeder2().getDirection());
+
+            ConnectablePosition connectablePositionTwt3 = twt3.getExtension(ConnectablePosition.class);
+            assertNotNull(connectablePositionTwt3);
+            connectablePositionTwt3.getFeeder1().setName("TWT3Modified.1", 1).setOrder(30, 1).setDirection(ConnectablePosition.Direction.BOTTOM, 1);
+            connectablePositionTwt3.getFeeder2().setName("TWT3Modified.2", 2).setOrder(30, 2).setDirection(ConnectablePosition.Direction.BOTTOM, 2);
+            connectablePositionTwt3.getFeeder3().setName("TWT3Modified.3", 3).setOrder(30, 3).setDirection(ConnectablePosition.Direction.BOTTOM, 3);
+            assertEquals("TWT3Modified.1", connectablePositionTwt3.getFeeder1().getName().orElseThrow());
+            assertEquals(30, connectablePositionTwt3.getFeeder1().getOrder().orElseThrow().intValue());
+            assertEquals(ConnectablePosition.Direction.BOTTOM, connectablePositionTwt3.getFeeder1().getDirection());
+            assertEquals("TWT3Modified.2", connectablePositionTwt3.getFeeder2().getName().orElseThrow());
+            assertEquals(30, connectablePositionTwt3.getFeeder2().getOrder().orElseThrow().intValue());
+            assertEquals(ConnectablePosition.Direction.BOTTOM, connectablePositionTwt3.getFeeder2().getDirection());
+            assertEquals("TWT3Modified.3", connectablePositionTwt3.getFeeder3().getName().orElseThrow());
+            assertEquals(30, connectablePositionTwt3.getFeeder3().getOrder().orElseThrow().intValue());
+            assertEquals(ConnectablePosition.Direction.BOTTOM, connectablePositionTwt3.getFeeder3().getDirection());
```

Comment:
Here, you should flush the network with : service.flush(network);

And then add some code to reload the network and test that the extension modifications have successfully been made in database (see other test methods like  loadDetailExtensionTest, voltagePerReactivePowerControlTest, slackTerminalExtensionTest ...)